### PR TITLE
[SU-97, SU-92] Development [SU-96] Categories list page and SU-92 Development [SU-91] Category page

### PR DESCRIPTION
**Pull request by UvgenGen**
[SU-97](https://youtrack.raccoongang.com/issue/SU-97) Development [SU-96] Categories list page
[SU-92](https://youtrack.raccoongang.com/issue/SU-92) Development [SU-91] Category page

### CODE REVIEW

**Review by madeira — APPROVED** [review]

File: lms/templates/category.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<%!
+from django.utils.translation import ugettext as _, get_language
+from django.core.urlresolvers import reverse
+%>
+<%page args="category" expression_filter="h"/>
+<article class="course" id="${category.id}" role="region" aria-label="${category.name}">
+    <a href="${reverse('courses_list_for_category', kwargs={'slug': category.slug})}">
+        <header class="course-image">
+            <div class="cover-image">
+                <img src="${category.img.url}" alt="${category.name}" />
```
Comment (sendr) [inline lms/templates/category.html]: if there is no `category.img` error will occure

File: lms/templates/course_category_list.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                  % if courses:
+                    <li class="courses-listing-item">
+                      <%include file="category.html" args="category=category" />
+                    </li>
+                  % endif
+                % endfor
+              </ul>
+            </section>
+        % else:
+          <h1>There are no categories on the platform.</h1>
```
Comment (sendr) [inline lms/templates/course_category_list.html]: wrap it please into `_()`

File: lms/templates/category.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    if category.img:
+        category_img_url = category.img.url
+    else:
+        category_img_url = ''
+%>
+<article class="course" id="${category.id}" role="region" aria-label="${category.name}">
+    <a href="${reverse('courses_list_for_category', kwargs={'slug': category.slug})}">
+        <header class="course-image">
+            <div class="cover-image">
+                <img src="${category_img_url}" alt="${category.name}" />
```
Comment (sendr) [inline lms/templates/category.html]: `${category.img.url if category.img else '' }`

File: lms/templates/courses_list_for_category.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+<main id="main" aria-label="Content" tabindex="-1">
+    <section class="home">
+    <%
+      courses = set(category.courses.all())
+    %>
+      <section class="course-box">
+        <h2 class="course-box__heading">${category.name}</h2>
+        <div>
+          <p class="course-box__description">${category.description}</p>
+          <%include file="${courses_list}" args="courses=courses" />
```
Comment (sendr) [inline lms/templates/courses_list_for_category.html]: `<%include file="${courses_list}" args="courses=category.courses.all()" />` will it work

**Review by sendr — APPROVED** [review]

**Pull request merged**